game/scripts/vscripts/gamemode.lua

```diff
@@ -129,6 +132,9 @@ function GameMode:OnGameInProgress()
   -- initialize modules
   InitModule(PointsManager)
   InitModule(CreepCamps)
+
+  -- start think timer
+  Timers:CreateTimer(0, Dynamic_Wrap(GameMode, "GameModeGoldThink"))
```

Comment:
You should create an initialization method inside of gold.lua and put both this and the playertable definition in there. The table should also be named something like "gold" since there's nothing aaa specific about it.